scripts/check-db-exists.ts

```diff
@@ -0,0 +1,58 @@
+import dotenv from 'dotenv'
+import cli from 'cli'
+import { Sequelize } from 'sequelize'
+import { spawn } from 'child_process'
+
+dotenv.config()
```

Comment:
Currently we only run dotenv.config if process.env.KUBERNETES is not true, i.e. not in a production environment where all the ENV_VARS are supposed to be set directly rather than provided by a .env file. `.env` is in the .dockerignore, so dotenv.config should normally do nothing, but for consistency maybe we should only conditionally run it on KUBERNETES !== true here as well.

Reply:
gotcha, i will also use dotenv-flow and copy the dotenv configuration params from appconfig.ts for consistency